.github/ISSUE_TEMPLATE.md

```diff
@@ -1,20 +1,28 @@
-## Version
-[provide output of the `$ openshift version` or `$ oc version` command]
+[provide a description of the issue]
 
-## Description
+##### Version
 
-## Steps To Reproduce
+[provide output of the `openshift version` or `oc version` command]
+
+##### Steps To Reproduce
 
 1. [step 1]
 2. [step 2]
 
-## Current Result
+##### Current Result
+
+...
```

Comment:
if we're going to bikeshed, i'd rather not have "..." here since everyone will have to delete it.

(same below)